packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -248,7 +248,7 @@ export default class ConfigParser {
             }
         }
 
-        return this.filterSpecs(specs, <string[]>exclude)
+        return this.filterSpecs([...spec, ...specs], <string[]>exclude)
```

Comment:
I think this change is unrelated to the issue, we can revert it which also should make the unit test pass.